packages/app/src/components/App.js

```diff
@@ -81,7 +81,12 @@ export class App extends Component {
                     getParentGraphMapFromVisualization(AO)
                 )
 
-                this.props.setVisualization(AO)
```

Comment:
The description only mentions that the fix is about updating the Analytics version. @edoardo could you explain why the `setVisualization` was removed? Is it being replaced by `clearVisualization` / `clearCurrent`?